src/player_activity.cpp

```diff
@@ -127,6 +127,7 @@ std::string player_activity::get_str_value(size_t index, std::string def) const
 
 void player_activity::do_turn( player *p )
 {
+    float season_mult = 91.0/calendar::turn.season_length();
```

Comment:
even so, add a constant to calendar to make it so you don't have to change it all over. :-)